sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/ConditionalAccessException.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.aad.webapp;
+
+import com.azure.spring.autoconfigure.aad.Constants;
+
+/**
+ * Create ConditionalAccessException to handle conditionalAccess in obo flow.
+ */
+public final class ConditionalAccessException extends RuntimeException {
```

Comment:
I think we need more content in java doc to explain how we use `ConditionalAccessException` to handle condition access in obo flow.

Reply:
Just like `Flux`:
![image](https://user-images.githubusercontent.com/13167207/104527791-84275d00-5640-11eb-97bc-19280f47973f.png)